experimental/dds/tree2/src/feature-libraries/sequence-field/sequenceFieldChangeEncoder.ts

```diff
@@ -11,132 +12,123 @@ import { Changeset, Mark } from "./format";
 import { isSkipMark } from "./utils";
 
 export const sequenceFieldChangeCodecFactory = <TNodeChange>(childCodec: IJsonCodec<TNodeChange>) =>
-	makeCodecFamily<Changeset<TNodeChange>>([
-		[
-			0,
-			{
-				encode: (changeset) => encodeForJson(changeset, childCodec),
-				decode: (changeset) => decodeJson(changeset, childCodec),
-			},
-		],
-	]);
+	makeCodecFamily<Changeset<TNodeChange>>([[0, makeV0Codec(childCodec)]]);
 
-function encodeForJson<TNodeChange>(
-	markList: Changeset<TNodeChange>,
+function makeV0Codec<TNodeChange>(
```

Comment:
i recommend you review this file with 'hide whitespace' enabled in diff options.